pricewars_merchant.py

```diff
@@ -143,6 +142,9 @@ def sold_offer(self, offer: SoldOffer) -> None:
         This method is called whenever the merchant sells a product.
         """
         print('Product sold')
+        self.inventory_level -= offer.amount_sold
```

Comment:
I want to avoid a fallback implementation to keep the example merchant simple.
We could either:
* Require the merchant to be reachable by the marketplace and have instant restocks (this implementation)
* or restock at the start of the next decision interval (previous implementation)

Reply:
Hmm ... not a fan of any of these solutions. :-/